research/src/pages/documentation/get-involved.mdx

```diff
@@ -114,27 +113,24 @@ Peer review is a building block of consensus. Even though you’ve already gotte
 
 #### What makes a good review
 
-Good external review takes time, kindness, and empathy. You’ll be reaching out to people who you may or may not have met, asking for input, waiting for them to be available, responding with enthusiasm,
-incorporating their feedback, and checking if you got it right.
+Good external review takes time, kindness, and empathy. You’ll be reaching out to people who you may or may not have met, asking for input, waiting for them to be available, responding with enthusiasm, incorporating their feedback, and checking if you got it right.
 
 #### How to get review
 
 Look up the W3C group you need review from, find the chair or another administrator, email them, and ask for advice. Or, if you don’t feel comfortable doing that, you can ask Greg (gwhitworth@salesforce.com) or Simon (simon@bocoup.com) to talk through who to talk to, and make an introduction. We recommend getting input from the following groups:
 
-- ARIA: Add an item to the ARIA WG weekly meeting agenda. Attend the ARIA weekly meeting and share your component accessible interaction model. G
-  et input.
+- ARIA: Add an item to the ARIA WG weekly meeting agenda. Attend the ARIA weekly meeting and share your component accessible interaction model. Get input.
 
   <p class="todo">
     Add some guidance on how to develop the accessibility model of a new control based on prior art
-    in ARIA. Also add information on where to raise issies, e.g: authoring practices task force -
+    in ARIA. Also add information on where to raise issues, e.g: authoring practices task force -
```

Comment:
a new issue should be created for this so it can be worked on.  @smhigley this seems something we could take on

Reply:
@scottaohara - Understand the intent of the comment. Otherwise, no additional changes seem to be required here. Does this otherwise need to delay this particular PR

Reply:
nope. i've already opened a new issue.  this can be resolved